ui: add isolation level information to SQL activity pages

jasonlmfong opened the pull request:
Introduce a column for 'Isolation Level' to the
active stmts/txns details pages, as well as the sessions page.

The change is achieved by augmenting the query to `crdb_internal.cluster_locks`
table via the SQL api in `/api/v2/`.

closes: [https://github.com/cockroachdb/cockroach/issues/152688](https://github.com/cockroachdb/cockroach/issues/152688)

Epic: None
Release note (ui change): A new section, 'Isolation Level' has
been added to the active statement and transaction details page and session page.

jasonlmfong committed:
docs: fix a typo in the documentation for sql stats

Epic: None
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/153617)
<!-- Reviewable:end -->

dhartunian reviewed (commented):
> @dhartunian reviewed 1 of 1 files at r1, 4 of 11 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/153617)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @alyshanjahani-crl)

___
*[`.github/copilot-instructions.md` line 1 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/153617#-O_JF8Pk8x8NHCMSR7F8:-O_JF8Pk8x8NHCMSR7F9:b-b33orm) ([raw file](https://github.com/cockroachdb/cockroach/blob/73a7218f5ffbbf4cc5beb87fe80cdfc7eec6dd2b/.github/copilot-instructions.md#L1)):*
> ```Markdown
> # AI Agent Instructions for CockroachDB
> ```

Remove from commit :smile: 
___
*[`pkg/ui/workspaces/cluster-ui/src/activeExecutions/execTableCommon.tsx` line 157 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/153617#-O_JFMAUAfwzS6B6I_0s:-O_JFMAUAfwzS6B6I_0t:b5nju38) ([raw file](https://github.com/cockroachdb/cockroach/blob/73a7218f5ffbbf4cc5beb87fe80cdfc7eec6dd2b/pkg/ui/workspaces/cluster-ui/src/activeExecutions/execTableCommon.tsx#L157)):*
> ```XML
>       placement="bottom"
>       style="tableTitle"
>       content={<p>The level of isolation the transaction was in.</p>}
> ```

This string reads oddly. I think it should be something like just "Isolation Level".


<!-- Sent from Reviewable.io -->

jasonlmfong commented on pkg/ui/workspaces/cluster-ui/src/sessions/sessionsTable.tsx:
```diff
@@ -254,6 +254,13 @@ export function makeSessionsColumns(
       cell: session => session.session.application_name,
       sort: session => session.session.application_name,
     },
+    {
```
makes sense

dhartunian reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/153617)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @alyshanjahani-crl and @jasonlmfong)

___
*[`pkg/ui/workspaces/cluster-ui/src/activeExecutions/execTableCommon.tsx` line 157 at r8](https://reviewable.io/reviews/cockroachdb/cockroach/153617#-O_SbWCH6U_qCt4PX2jD:-O_SbWCH6U_qCt4PX2jE:b-mmb8yx) ([raw file](https://github.com/cockroachdb/cockroach/blob/f1109fe25c95a320182a943f26ac0d9b48e57071/pkg/ui/workspaces/cluster-ui/src/activeExecutions/execTableCommon.tsx#L157)):*
> ```XML
>       placement="bottom"
>       style="tableTitle"
>       content={<p>Isolation level.</p>}
> ```

nit: tooltip text should be a complete sentence like the others above.
___
*[`pkg/ui/workspaces/cluster-ui/src/statsTableUtil/statsTableUtil.tsx` line 41 at r8](https://reviewable.io/reviews/cockroachdb/cockroach/153617#-O_ScQF4-b5TWZ3V-io3:-O_ScQF4-b5TWZ3V-io4:b-8wkdjl) ([raw file](https://github.com/cockroachdb/cockroach/blob/f1109fe25c95a320182a943f26ac0d9b48e57071/pkg/ui/workspaces/cluster-ui/src/statsTableUtil/statsTableUtil.tsx#L41)):*
> ```XML
>   mostRecentStatement: "Most Recent Statement",
>   networkBytes: "Network",
>   isolationLevel: "Isolation Level",
> ```

Given the change above, should we call this `Default Isolation Level`?


<!-- Sent from Reviewable.io -->

jasonlmfong commented:
> nit: tooltip text should be a complete sentence like the others above.
thanks, will fix

> Given the change above, should we call this Default Isolation Level?
I don't think so, this can be the non-default isolation as well

jasonlmfong commented:
> It should say "Default Isolation Level" b/c transactions within the session can set a different isolation level, but whatever isolation level is set at the session level is the "default isolation level"

ah i see what you mean, will update

jasonlmfong committed:
ui: add isolation level information to active statements/transactions/sessions

This change introduces a column for 'Isolation Level' to the
active stmts/txns details pages, as well as the sessions page.

The change is achieved by augmenting the query to `crdb_internal.cluster_locks`
table via the SQL api in `/api/v2/`.

closes: [#152688](https://github.com/cockroachdb/cockroach/issues/152688)
Epic: None
Release note (ui change): A new section, 'Isolation Level' has
been added to the active statement and transaction details page and session page.

jasonlmfong commented:
bors r+
